update Lifecycling docs (#429)
digital-asset/daml-finance: docs/source/concepts/lifecycling.rst

In this scenario we go through the process of paying a cash dividend from an issuer to an investor.
The initial state looks as follows:

* An issuer maintains an ``ACME`` instrument representing shares in a company
* An investor owns a holding of 1000 units of the ``ACME`` instrument (version 1) with the issuer

> * The issuer wants to process and pay a cash dividend of USD 10.0 per share of its ``ACME``

Reviewer: ```suggestion
* The issuer wants to process and pay a cash dividend of USD 10.0 per unit of its ``ACME``
```

   :alt: The investor claims the lifecycle effect through the claim rule, passing in their ACME v1
         holding. This produces a batch and settlement instructions.

Note that the party responsible for claiming an effect can be specified flexibly in the
:ref:`Claim Rule <type-daml-finance-interface-lifecycle-rule-claim-claim-29284>` contract. Through

> this custodians can be given the authority to push a given corporate action to the asset holder, as

Reviewer 2: ```suggestion
this contract, custodians can be given the authority to push a given corporate action to the asset holder as
```

         allocates a 1000 ACME v2 holding to the second instruction and a 10000 USD holding to the
         third.

> In other words, the investor receives a 10'000 USD dividend and 1'000 shares of ``ACME`` v2 in
> return for 1'000 shares of ``ACME`` v1.

Reviewer 2: ```suggestion
The result of processing the settlement batch results in the investor receiving a 10'000 USD dividend and 1'000 shares of ``ACME`` v2 in
return for their 1'000 shares of ``ACME`` v1.
```
